test/unit/lib/plugins/aws/package/lib/cleanupTemplateResources.test.js

```diff
@@ -42,4 +42,23 @@ describe('test/unit/lib/plugins/aws/package/lib/stripNullPropsFromTemplateResour
   it('Should not affect resources without null props', async () => {
     expect(Object.keys(finalTemplate.Resources.anotherBucket.Properties).length).to.equal(1);
   });
+
+  it('should remove empty resources for first policy document', async () => {
+    const result = await runServerless({
+      fixture: 'aws',
```

Comment:
Could we use the `function` fixture here instead? It will allow us to not extend the `aws` fixture with additional `index.js` with function implementation to keep it "plain".